Frameworks/UIKit/UIButton.mm

```diff
@@ -891,22 +946,18 @@ - (UIImageView*)imageView {
 }
 
 /**
- @Status Caveat
- @Notes Returns the receiving view
-*/
-- (UIView*)viewForBaselineLayout {
-    return self;
-}
-
-/**
  @Status Interoperable
 */
 - (CGSize)intrinsicContentSize {
     CGSize ret = CGSizeZero;
 
     UIImage* img = self.currentImage;
     UIEdgeInsets contentInsets = self.contentEdgeInsets;
-    CGSize textSize = [[self currentTitle] sizeWithFont:self.font];
+
+    // TODO: Should we be asking our label for its intrinsicContentSize?
```

Comment:
Oh,  i think i missunerstand your point earlier.

It is a little weird why we are not asking TitleLabel for its size, instead, we are using the sizeWithFont with that string. This does not make sense. some of properties on UIButton such as linkbreakMode is just a pass through to UILbael.

---
In reply to: [100653705](https://github.com/Microsoft/WinObjC/pull/1920#discussion_r100653705) [](ancestors = 100653705,100653270,100653027)